.github/workflows/centraldb_angular_docker_publish.yaml

```diff
@@ -9,12 +9,12 @@ on:
       - releasing/version/VERSION
 
 env:
-  DOCKER_USER: kubeflownotebookswg
-  IMG: kubeflownotebookswg/centraldashboard-angular
+  DOCKER_USER: kubeflow
```

Comment:
I'm almost sure that this is not needed anymore.

Reply:
Yes, the right term is anyway OCI images, not Docker images.

Reply:
@andyatmiami if you and other don't mind I can do this in a FUP PR. Just to try to make this merge as soon as possible (now that we have a green build).